### DISCUSSION TITLE
Penumbra behavior changed after reboot

### DISCUSSION THREAD

**Discussion by sgsoig** [xivdev/Penumbra#91] [Q&A]
Hello,

I booted up XIV and I noticed some changes in Penumbra's behavior. There was no update, but now when I import a mod, there is no progress bar like before. It just says import in progress. 
![image](https://user-images.githubusercontent.com/96228012/146327514-785e7f2c-eb38-4d59-9109-41a142a1bcb4.png)

Secondly, mods that worked in the past no longer work, Here I have modded some boots and the desired effect just no longer works? 
![image](https://user-images.githubusercontent.com/96228012/146327841-ee43844e-20ad-4fb2-8b91-669b4bef60cc.png)

These boots are suposed to look like this, and they do not. 
![image](https://user-images.githubusercontent.com/96228012/146327902-adb7292f-836b-4861-a058-e3034da688c3.png)

However, another set of boots from this same mod pack works.
![image](https://user-images.githubusercontent.com/96228012/146328080-c794e52f-c861-4e24-b19e-ded6a16a7d2d.png)

All this behavior changed from on boot to the other. Also, Penumbra lags now? I am not sure what caused this change

**Comment by sgsoig** [comment]
Hrm, well creating a back up and restarting and installing it solved all the issues? Still curious as to what could have caused this !

**Comment by Ottermandias** [comment]
Do you have any remnants of Penumbra in your devPlugins folder or something like that, maybe?

**Reply by sgsoig** [reply]
nope! I Installed Penumbra when it was only through the repo link, never the old install method
